entities/edge.go

```diff
@@ -0,0 +1,28 @@
+package entities
+
+// EdgeType is available type of edge on phabricator.
+type EdgeType string
+
+var (
+	EdgeCommitRevision EdgeType = "commit.revision"
```

Comment:
Might want to check what will happen if edge type is not one of mentioned... This might happen if phabricator is upgraded and new type is introduced... Wouldn't be nice to "crash"...

Reply:
This is for request only - feel free to ignore...